packages/app/src/systems/Chains/fuel/utils/provider.ts

```diff
@@ -0,0 +1,38 @@
+import type { FetchRequestOptions } from 'fuels';
+import { Provider } from 'fuels';
+
+let requestTimestamps: number[] = [];
+const maxRequestsPerSecond = 10;
```

Comment:
we can increase right now the limit is 30

Reply:
I'll increase to 20 as I want to have some room because the wallet uses its own provider and can do requests out of created provider scope